src/cart/crt_context.c

```diff
@@ -635,39 +700,9 @@ crt_context_destroy(crt_context_t crt_ctx, int force)
 	if (!force && rc && i == CRT_SWIM_FLUSH_ATTEMPTS)
 		D_GOTO(err_unlock, rc);
 
-	rc = d_hash_table_destroy_inplace(&ctx->cc_epi_table, true /* force */);
-	if (rc) {
-		D_ERROR("destroy context (idx %d, force %d), "
-			"d_hash_table_destroy_inplace failed, rc: %d.\n",
-			ctx->cc_idx, force, rc);
-		if (!force)
-			D_GOTO(err_unlock, rc);
-	}
-
-	d_binheap_destroy_inplace(&ctx->cc_bh_timeout);
-
 	D_MUTEX_UNLOCK(&ctx->cc_mutex);
```

Comment:
Noticed a bug in this patch: `crt_context_destroy` acquires `cc_mutex` and `epi_mutex` in the wrong order, and `crt_ctx_epi_abort` should not release and acquire `cc_mutex`.

Reply:
Fixed.